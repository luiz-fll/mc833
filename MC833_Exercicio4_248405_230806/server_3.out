remoto: 127.0.0.1:6003
[SERVIDOR] Escutando em 127.0.0.1:6003
127.0.0.1:35114
GET / HTTP/1.0
Host: teste

127.0.0.1:35126
GET / HTTP/1.0
Host: teste

127.0.0.1:35132
GET / HTTP/1.0
Host: teste

127.0.0.1:35140
GET / HTTP/1.0
Host: teste

127.0.0.1:35148
GET / HTTP/1.0
Host: teste

127.0.0.1:35164
GET / HTTP/1.0
Host: teste

127.0.0.1:35178
GET / HTTP/1.0
Host: teste

127.0.0.1:35188
GET / HTTP/1.0
Host: teste

127.0.0.1:35202
GET / HTTP/1.0
Host: teste

