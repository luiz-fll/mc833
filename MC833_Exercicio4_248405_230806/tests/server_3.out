remoto: 127.0.0.1:50003
[SERVIDOR] Escutando em 127.0.0.1:50003
127.0.0.1:47628
GET / HTTP/1.0
Host: teste

127.0.0.1:47644
GET / HTTP/1.0
Host: teste

127.0.0.1:47650
GET / HTTP/1.0
Host: teste

127.0.0.1:47656
GET / HTTP/1.0
Host: teste

127.0.0.1:47668
GET / HTTP/1.0
Host: teste

127.0.0.1:47674
GET / HTTP/1.0
Host: teste

127.0.0.1:47688
GET / HTTP/1.0
Host: teste

127.0.0.1:47700
GET / HTTP/1.0
Host: teste

127.0.0.1:47714
GET / HTTP/1.0
Host: teste

127.0.0.1:47724
GET / HTTP/1.0
Host: teste

